Source/DataSources/EntityView.js

```diff
@@ -317,9 +317,6 @@ define([
             var hasViewFrom = defined(viewFromProperty);
 
             if (!hasViewFrom && defined(boundingSphere)) {
-                var controller = scene.screenSpaceCameraController;
-                controller.minimumZoomDistance = Math.min(controller.minimumZoomDistance, boundingSphere.radius * 0.5);
```

Comment:
@mramato If no minimumZoom is specified, it goes to 1, and the flyTo will go to about 100 units away. But this problem is fixed by the clamping of the zoom to the `screenSpaceCameraController` minimum and maximum zoom.